### ISSUE TITLE
Simplify listings page

### ISSUE THREAD

**Issue by ajefts** [topcoder-platform/community-app#852]
Let's reduce the complexity of the default view of the challenge listings page (https://www.topcoder.com/challenges/).

1. Remove "Ongoing Challenges" and "Past Challenges" from the default view.  i.e. Leave the links to the views in the right nav, but don't load the buckets of challenges.  This will reduce the default listings view to My and Open for Registration challenges.

**Comment by ajefts** [comment]
@birdofpreyru Can you run a quick challenge for this one?  Are there any other "quick win" improvements we can do to the page to improve the loading experience?

**Comment by birdofpreyru** [comment]
@ajefts the number of buckets does not affect the loading time. Do you still want to remove *Ongoing Challenges* and *Past Challenges* buckets then?

A quick improvement would be to remove or reconfigure API rate limits. At the moment, frontend sequentializes and throttles all requests API to guarantee no more than 5 requests per second. I enter the listing, there are 13 API requests sent; thus ~2.5 seconds delay just because of this throttling. But without the throtting people were hitting API rate limits now and then and complaining about those errors.

**Comment by ajefts** [comment]
@birdofpreyru Why wouldn't it improve performance without those sections?  Does the page still need to do all the API calls and draw those sections if they aren't there?

For rate limiting, let's talk to @mtwomey about how to be smarter about our limits.

**Comment by birdofpreyru** [comment]
@ajefts it does not pull challenges by sections. It preloads all active challenges, and then use them for any section that is based on active challenges. Removing *Past Challenges* will save some calls, but they are anyway not blocking (only the *Past Challenges* bucket waits for loading of past challenges, others don't).

**Comment by ajefts** [comment]
Ok, I'd still like to start by removing the past challenge section then.
It shoudl only load if I click the nav for it.  I think it'll help visually
for users to consume the page. There is a lot to look at there now....

**Comment by birdofpreyru** [comment]
@ajefts ok, the *Past Challenges* bucket is removed. It was necessary just to comment out a single line, so I did myself. On the way, noticed that challenge icon in the listing were empty. It turned out, that at some point API switched from returning subTrack values as all-caps with underscores to human-readable names, and that broke the icons (luckily, not entire page). Fixed that as well, by normalizing subtrack names at community-app side, but it is definitely not good that our dev practices allow to roll out such change in API without anybody noticing, until somebody notice a change caused by that somewhere in the frontend :(
